opencti-platform/opencti-graphql/src/manager/retentionManager.js

```diff
@@ -84,7 +84,7 @@ const retentionHandler = async () => {
 const initRetentionManager = () => {
   let scheduler;
   return {
-    start: () => {
+    start: async () => {
```

Comment:
ditto, this change seems unrelated to error/log management, is it intended ?

Reply:
Its unrelated, just a warning fix